input/ig.xml

```diff
@@ -632,31 +632,31 @@
 				<reference value="PractitionerRole/bobrester-bob-gp"/>
 			</reference>
 			<name value="PractitionerRole - General practitioner"/>
-			<description value="Dr. Bob Bobrester, GP, at Bobrester Medical Centre"/>
+			<description value="Shows an example of a General practitioner for the *AU Core PractitionerRole* profile with practitioner and organisation as references."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitionerrole"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="PractitionerRole/cardiologist-sallie-sutherland"/>
 			</reference>
 			<name value="PractitionerRole - Cardiologist"/>
-			<description value="Dr. Sallie Sutherland, Murrabit Public Hospital"/>
+			<description value="Shows an example of a Cardiologist for the *AU Core PractitionerRole* profile that demonstrates the use of Medicare Provider Number and practitioner, organization and location references."/>
```

Comment:
1. "Cardiologist" to be "cardiologist"
2. Title case the FHIR resources.

Reply:
Done